lib/src/core/client/impl/tezart_client.dart

```diff
@@ -55,19 +59,34 @@ class TezartClient {
             amount: amount,
             destination: destination,
             customFee: customFee,
+            gasLimit: gasLimit,
+            storageLimit: storageLimit,
           ),
         );
-      if (reveal) await _prependRevealIfNotRevealed(operationsList, source);
+      if (reveal) {
+        await _prependRevealIfNotRevealed(
+          operationsList,
+          source,
+          customFee,
+          gasLimit,
+          storageLimit,
+        );
+      }
 
       return operationsList;
     });
   }
 
-  /// Returns an [OperationsList] that reveals [source.publicKey]
-  OperationsList revealKeyOperation(Keystore source) {
+  /// Returns an [OperationsList] that reveals [source] public key.
+  OperationsList revealKeyOperation(Keystore source, [int? customFee, int? gasLimit, int? storageLimit]) {
```

Comment:
Idéalement, il faudrait utiliser des named paramètres pour les options:
```suggestion
  OperationsList revealKeyOperation(Keystore source, {int? customFee, int? gasLimit, int? storageLimit}) {
```